### ISSUE TITLE
Incorrect ternary parsing in rescript when using newline (rescript)

### ISSUE THREAD

**Issue by giraud** [giraud/reasonml-idea-plugin#424]
plugin version: 0.115

### Description

with this code 
```rescript
let fn = x => x
  ? true
  : false
```

function body is not parsed correctly
![image](https://github.com/giraud/reasonml-idea-plugin/assets/239612/d0a6007c-aa7d-4275-8e89-7a5f766c1cfe)

**Comment by giraud** [comment]
```rescript
let fn = x => x
  ? "a"
  : "b" ++ 
    "c"
```

**Comment by giraud** [comment]
Fixed in 0.116
